src/IconComponents/Twitter.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 
 const Twitter = (props) => {
   return (
-    <svg version="1.1" x="0px" y="0px" viewBox="0 0 1792 1792">
-      <path fill={props.fill} d="M1408 610q-56 25-121 34 68-40 93-117-65 38-134 51-61-66-153-66-87 0-148.5 61.5t-61.5 148.5q0 29 5 48-129-7-242-65t-192-155q-29 50-29 106 0 114 91 175-47-1-100-26v2q0 75 50 133.5t123 72.5q-29 8-51 8-13 0-39-4 21 63 74.5 104t121.5 42q-116 90-261 90-26 0-50-3 148 94 322 94 112 0 210-35.5t168-95 120.5-137 75-162 24.5-168.5q0-18-1-27 63-45 105-109zm256-194v960q0 119-84.5 203.5t-203.5 84.5h-960q-119 0-203.5-84.5t-84.5-203.5v-960q0-119 84.5-203.5t203.5-84.5h960q119 0 203.5 84.5t84.5 203.5z"/>
-    </svg>
+    <svg version="1.1" x="0px" y="0px" viewBox="0 0 1792 1792" style={{ backgroundColor: "skyblue" }} >
```

Comment:
You should use a hex for color - it enables you to be much more specific. For example, Twitter blue is `#1da1f2`. https://www.lockedowndesign.com/social-media-colors/